db_json_util.py

```diff
@@ -74,6 +76,44 @@ def main() -> None:
                     else:
                         print(response.text)
 
+# export data using the API
+        if args.export_data:
+            # check if path exists
+            if args.json_chains:
+                if Path.cwd() / args.json_chains:
```

Comment:
This already exists as a variable, `path_chains`. Also, this line as it stands now only checks if the variable isn't `Null` but I think what you want to do is to check if there exists a file on the path:

```suggestion
                if path_chains.exists():
```